smartcontactmanager/src/main/java/com/scm/smartcontactmanager/dao/category/CategoryDaoImpl.java

```diff
@@ -1,8 +1,80 @@
 package com.scm.smartcontactmanager.dao.category;
 
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Repository;
 
+import com.scm.smartcontactmanager.entity.category.Category;
+import com.scm.smartcontactmanager.entity.contact.Contact;
+import com.scm.smartcontactmanager.entity.contactcategoryrelation.ContactCategoryRelation;
+import com.scm.smartcontactmanager.repository.category.CategoryRepository;
+import com.scm.smartcontactmanager.repository.contact.ContactRespository;
+import com.scm.smartcontactmanager.repository.contactcategoryrelation.ContactCategoryRelationRepository;
+
 @Repository
-public class CategoryDaoImpl implements CategoryDao{
+public class CategoryDaoImpl implements CategoryDao {
+	@Autowired
+	private CategoryRepository categoryRepository;
+	
+	@Autowired
+	private ContactRespository contactRespository ;
+	
+	@Autowired
+	private ContactCategoryRelationRepository contactCategoryRelationRepository ;
+	
+//	@Autowired
+//	private ContactCategoryRelationRepository contactCategoryRelationRepository;
+
+	@Override
+	public Category createCategory(Category category) {
+		Category saveContact = categoryRepository.save(category);
+		return saveContact;
+	}
+	@Override
+	public List<Category> getListOfCategory(){
+		return categoryRepository.findAll();
+		
```

Comment:
Remove extra new lines where ever application

Reply:
The issue have been resolved. Please check it once.